### ISSUE TITLE
Changing circle indicator color

### ISSUE THREAD

**Issue by krhckd93** [ongakuer/CircleIndicator#82]
Is it possible to change the color of the circles or the dots?

**Comment by trihung93** [comment]
Just create an XML with your colors and use it with attrs :  ci_drawable + ci_drawable_unselected
```

<shape xmlns:android="http://schemas.android.com/apk/res/android"
    android:shape="oval">
    <solid
        android:color="@color/primary"/>
</shape>

app:ci_drawable="@drawable/circle_radius_selected"
 app:ci_drawable_unselected="@drawable/circle_radius_unselected"
```

**Comment by tonroitman** [comment]
Is there a way to change it programmatically?

**Comment by trihung93** [comment]
@tonroitman : you can edit this lib, you should add method Get and Set for selectedIndicator/currentIndicator.
create 2 drawable for selectedIndicator and currentIndicator, after that, you can set Background Drawable.

selectedIndicator.setBackgroundDrawable / selectedIndicator.setBackground
currentIndicator.setBackgroundDrawable / currentIndicator.setBackground

**Comment by Mariovc** [comment]
Someone forked this lib to make this feature? I would like to change color dynamically (e.g. random color each second)

**Comment by chihung93** [comment]
@Mariovc  : Just do it yourself .

**Comment by deka91** [comment]
@Mariovc : can you send me the link for this?

**Comment by Mariovc** [comment]
https://github.com/ryanwr/CircleIndicator/commit/da4461bfc36d57ea705d1731b5aa5f475fc2f190

**Comment by LamourBt** [comment]
This issue was closed but it seems that [da4461b](https://github.com/ryanwr/CircleIndicator/commit/da4461bfc36d57ea705d1731b5aa5f475fc2f190) was never merged into the main branch. Currently, it is impossible to change both the selected and unselected drawable programmatically.
